python-sdk/docs/astro/sql/operators/append.rst

```diff
@@ -7,7 +7,7 @@ When to use the ``append`` operator
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The ``append`` operator allows you to append data from a source table to a target table.
 
-.. literalinclude:: ../../../../example_dags/example_append.py
```

Comment:
@feluelle why are we nesting these DAGs instead of just having them as separate folders? We are already filling a DAGBag manually so it wouldn't be too hard to just point based on an env variable https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/test_example_dags.py#L54

We could even have it take a list so if we expect 2.4 to run the 2.3 and 2.2.5 DAGs we just have the 2.4 run take the argument `TEST_AIRFLOW_VERSIONS=2.4,2.3,2.2.5`. 

This structure would be confusing to users.